ci.gradle

```diff
@@ -56,6 +55,26 @@ gitPublish {
 }
 gitPublishCopy.dependsOn(scaladoc)
 
+task provision(type: Exec) {
```

Comment:
Just wondering, what do you see as an advantage of having this as a gradle task and not a shell script checked in git? That way it's easier to edit or even e.g. use shellcheck... If we want to have whole lifecycle in gradle, we can call the script from gradle task, I guess... ?

Reply:
I had it as a script but did not really know where to put it and thought it nicer to have everything in Gradle. I'm happy to move it into a file. Oh, one reason is also the Mesos version.

Reply:
I find it pretty hard to read and edit, since it's just plain string... if it was up to me, I would create a folder similar to one we have in marathon and place it there. But let's see what others think...

Reply:
So, `./ci/provision.sh 1.7.0`?

Reply:
moving out to `sh` file sounds good.   there is still value in having a gradle task to invoke there right?  a precondition of the task could be the exists of the installation making this part of an incremental build.